gen/generate.go

```diff
@@ -234,7 +243,9 @@ func generateModule(m *compile.Module, i thriftPackageImporter, builder *generat
 		}
 
 		// TODO(abg): Verify no file collisions
-		files["constants.go"] = buff.Bytes()
+		if !o.NoConstants {
+			files["constants.go"] = buff.Bytes()
```

Comment:
Maybe do this in the `if` check above:

    if !o.NoConstants && len(m.Constants) > 0 {
      ..